src/Components/Facility/FacilityCreate.tsx

```diff
@@ -984,7 +983,7 @@ export const FacilityCreate = (props: FacilityProps) => {
             >
               <button
                 className="btn btn-default border-2 border-grey-300"
-                onClick={() => goBack()}
+                onClick={() => navigate(`/facility/${facilityId}`)}
```

Comment:
@cp-Coder can you try event.preventDefault(), that should fix this, if that doesn't work then let's try hard coding the navigation path